.nvmrc

```diff
@@ -0,0 +1 @@
+v10
```

Comment:
Hi @AdrieanKhisbe, do we need .nvmrc? 

Also, question to everyone: should we stick with any particular node version? I always used LTS versions and never used current for example.

Reply:
Hi @rpeshkov =)

I'm not sure. the reason I did is to avoid using different version of version of npm which result into modification of the package-lock.json. (use of ^caret lock)

And indeed, it's better to stick to 8 LTS, which also use now npm 6.

I wait confirmation from others, and I'll make that change :)

Reply:
I'm not particularly aware about node versions but we should support the LTS **and** the current, not?

Reply:
Prerequisites for VSCode development contain  LTS, so I guess there's no point in supporting and testing over current version. https://github.com/Microsoft/vscode/wiki/How-to-Contribute#build-and-run

Reply:
VScode use the electron node version: (https://github.com/Microsoft/vscode/issues/11660) currently 8.9.3 (from the about dialog).

But what .nvmrc does and what "node_js" part from ".travis.yml" does ? it seems redundant…

Reply:
It's not redundant. 

- nvmrc is for development purpose, where we should use the latest.
- nodejs block in travis is for the range of node we are testing against

Reply:
Hi guys, my 2 cents here:

1. I would stick with .travis.yml file, that's in documentation for extensions development: https://code.visualstudio.com/docs/extensions/testing-extensions#_running-tests-automatically-on-travis-ci-build-machines
2. It's better to use node of the same branch as bundled in VSCode. All the extensions are run on the node version that's bundled: https://github.com/Microsoft/vscode/issues/18253#issuecomment-271235914
If we will be using current version and not LTS, there might be some compatibility issues, different APIs and many other things.

Reply:
@AdrieanKhisbe so, if I understand Travis doesn't care about .nvmrc…
@rpeshkov 
1. According to https://code.visualstudio.com/docs/extensions/testing-extensions#_running-tests-automatically-on-travis-ci-build-machines, if we want to check the extension against different versions of VSCode we should use the CODE_VERSION environment variable. I think the "node_js" part of the travis file is useless because the previous link say the script "download and unzip VS Code" so it uses the bundled node version in any case…
2. Yes, we should use LTS in development, because it's the branch used by VSCode.

Reply:
@sipi Yes indeed, it's only for development purposes

I'll remove the node_js section as you agree that you think it's useless